ci/docker-rust-nightly/Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM solanalabs/rust:1.43.0
+FROM solanalabs/rust:1.44.0
 ARG date
 
 RUN set -x \
  && rustup install nightly-$date \
  && rustup component add clippy --toolchain=nightly-$date \
+ && rustup component add rustfmt --toolchain=nightly-$date \
```

Comment:
Well, this is needed: https://buildkite.com/solana-labs/solana/builds/26037#29954e66-85ab-4c7b-b90a-1454aae6d52e/82-111